universum.cabal

```diff
@@ -85,3 +85,18 @@ library
 
   hs-source-dirs:      src
   default-language:    Haskell2010
+
+benchmark universum-benchmark
+  type:                exitcode-stdio-1.0
+  default-language:    Haskell2010
+  ghc-options:         -Wall -O2 -threaded -rtsopts -with-rtsopts=-N
+  hs-source-dirs:      benchmark
+  main-is:             Main.hs
+  build-depends:
+    base                  >= 4.7    && <4.11,
```

Comment:
Current formatting of `build-depends` section is not really convenient... And version policy is not specified. @vrom911 could you also reformat `.cabal` file and dependencies after I write what to do in Issue #54?

Reply:
Sure. I tried to make .cabal format similar to what already was there, but it's not in uniform style.. Will do this after benchmarks, thanks!